src/implementations/vanilla/components/text-area/_text-area.scss

```diff
@@ -0,0 +1,110 @@
+/**
+ * Textarea component
+ * @define text-area
+ */
+
+@use "sass:map";
```

Comment:
Updated and verify for text field, same spacings 👍

Reply:
isn't this still the case? in EU the text-area margin-top is supposed to be 0.5rem, 8px, i see 12px

Reply:
It seems to me that his H associated in EC to red color of the "invalid text" on hover means red hover, the same applied to the textarea when hovering it, but also in EU the color on hover should change consistently in the invalid text and the border of the form input
![image](https://user-images.githubusercontent.com/14326885/107755157-1ea3ba80-6d2b-11eb-8a56-afe9bc68152e.png)

Reply:
Swapped the form element top margin 👍 
Not sure to understand your last comment for the hover color of the invalid text. after checking the border colors correspond to the defined specs.

Reply:
the border yes, but in the specs i see also the invalid text changing color consistently on hover, which is not happening in our implementation, also because we trigger the hover status of the text-area only. So, basically, it looks like this behaviour could be standardised in the form-group css, so that when hovering the form-group container both the input border and the invalid text would get the same, different color.